types/shelljs/tslint.json

```diff
@@ -1,7 +1,17 @@
 {
-    "extends": "dtslint/dt.json",
-    "rules": {
-        "max-line-length": false,
-        "unified-signatures": false
-    }
+	"extends": "dtslint/dt.json",
+	"rules": {
+		"indent": [
+			true,
+			"tabs"
```

Comment:
Disk space savings and personal preference, since I’m&nbsp;effectively re&#x2011;writing the&nbsp;type definition, I’m&nbsp;also converting it&nbsp;to&nbsp;use&nbsp;tabs, which take one byte instead of&nbsp;four.

Reply:
OK. Not a hill I will die on (and also not sure what this project's guidelines/conventions are - if any).
I just noticed that the few other random typings I checked (for JS packages) all used spaces (which is objectively much better than tabs 😛 :trollface:)

I definitely prefer spaces (and they render much nicer on GitHub too), but I don't mind keeping the tabs either 🤷‍♂️

Reply:
DT lets authors decide on a per-package basis so either is fine.

(but typescript-bot secretly prefers spaces :robot: )